src/json/members/benxu.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Benjamin Xu",
+    "pennkey": "benxu",
+    "major": "CIS",
+    "school": "SEAS",
+    "bio": "Hi I'm Benjamin, a sophomore studying CIS in SEAS and working on Backend for OHQ.",
+    "hometown": "Saratoga, California",
+    "team": "OHQ",
```

Comment:
Could you please update the `team` name to match the other OHQ people json's

Reply:
done